opengrok-tools/src/test/python/test_commands.py

```diff
@@ -41,45 +41,45 @@ def test_str(self):
                         [{"command": ['foo']}, {"command": ["bar"]}]))
         self.assertEqual("opengrok-master", str(cmds))
 
-    @unittest.skipUnless(os.path.exists('/bin/true') and os.path.exists('/bin/false'), "requires Unix")
+    @unittest.skipUnless(os.path.exists('/bin/true') and os.path.exists('/bin/cp'), "requires Unix")
     def test_run_retcodes(self):
         cmds = Commands(CommandsBase("opengrok-master",
                         [{"command": ["/bin/echo"]},
                          {"command": ["/bin/true"]},
-                         {"command": ["/bin/false"]}]))
+                         {"command": ["/bin/ls"]}]))
         cmds.run()
         # print(p.retcodes)
         self.assertEqual({'/bin/echo opengrok-master': 0,
                           '/bin/true opengrok-master': 0,
-                          '/bin/false opengrok-master': 1}, cmds.retcodes)
+                          '/bin/ls opengrok-master': 2}, cmds.retcodes)
 
-    @unittest.skipUnless(os.path.exists('/usr/bin/true') and os.path.exists('/usr/bin/false'), "requires Unix")
+    @unittest.skipUnless(os.path.exists('/bin/true') and os.path.exists('/bin/ls') and os.path.exists('/bin/echo'), "requires Unix")
```

Comment:
it had a reason there was a /usr. this way it worked for me on mac.

Reply:
if you remove it then it runs just twice the same test

Reply:
on highSierra I see:
```
[
bash
cat
chmod
cp
csh
date
dd
df
domainname
echo
ed
expr
hostname
kill
ksh
launchctl
link
ln
ls
mkdir
mv
pax
ps
pwd
rm
rmdir
sh
sleep
stty
sync
tcsh
test
unlink
wait4path
zsh
```

Reply:
yes exactly, there's no `false` nor `true` in it. I wanted to run the test also on my machine and they're in `usr/bin`

Reply:
The tests need to use common denominator, duplication is wrong.